fenix/gradle.properties

```diff
@@ -27,3 +27,5 @@ libUrl=https://github.com/mozilla-mobile/firefox-android/tree/main/fenix
 libVcsUrl=https://github.com/mozilla-mobile/firefox-android.git
 libLicense=MPL-2.0
 libLicenseUrl=https://www.mozilla.org/en-US/MPL/2.0/
+android.nonTransitiveRClass=false
+android.nonFinalResIds=false
```

Comment:
> From Gradle 8 the default value has changed from false to true.

Does this mean that we are failing to use non-transitive R classes and non-final res Ids? Can we file a follow-up to fix this if that's true?

Reply:
Yes, we don't use them in Fenix or Focus. Moving to non-transitive R classes and non-final res Ids implies changing hundreds of files, so I thought it would be best not to do it here. And we still should decide if we want to.

Reply:
Sounds good, let's file a bug then so we can track if we want to do it or not. I'm not sure what either direction implies right now.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1863329 and https://bugzilla.mozilla.org/show_bug.cgi?id=1863330